src/components/Article.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import '../App.scss';
+
+export const Article = props => (
```

Comment:
Add prop types to the Article component
```suggestion
export const Article = (props) => (
```